From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: configure doesn't care about cross compiling

When cross compiling, we get this error:

  checking whether sys/inotify.h actually works... configure: error:
  cannot run test program while cross compiling

We assume that if somebody cross compiles, he made sure that the inotify
headers are there and working.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 configure.ac |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

Index: inotify-tools-3.13/configure.ac
===================================================================
--- inotify-tools-3.13.orig/configure.ac
+++ inotify-tools-3.13/configure.ac
@@ -50,7 +50,11 @@
                   [[return (-1 == inotify_init());]]
   ),
   [AC_MSG_RESULT([yup]); AC_DEFINE([SYS_INOTIFY_H_EXISTS_AND_WORKS],[1],[sys/inotify.h exists and works correctly])],
-  [AC_MSG_RESULT([nope, using own inotify headers])]
+  [AC_MSG_RESULT([nope, using own inotify headers])],
+  [
+	AC_MSG_RESULT([cross compiling, assuming yes]);
+	AC_DEFINE([SYS_INOTIFY_H_EXISTS_AND_WORKS],[1],[sys/inotify.h exists and works correctly])
+  ]
 )
 
 
